ingestion-beam/src/main/java/com/mozilla/telemetry/io/Read.java

```diff
@@ -46,8 +48,15 @@ public PubsubInput(ValueProvider<String> subscription) {
     @Override
     public Result<PCollection<PubsubMessage>> expand(PBegin input) {
       return input //
-          .apply(PubsubIO.readMessagesWithAttributes().fromSubscription(subscription))
-          .apply(ToPubsubMessageFrom.identity());
+          .apply(PubsubIO.readMessagesWithAttributesAndMessageId().fromSubscription(subscription))
+          .apply(MapElements.via(new SimpleFunction<PubsubMessage, PubsubMessage>() {
```

Comment:
We've generally been preferring to use lambdas where possible rather than anonymous subclasses, so this would be:

```
MapElements.into(TypeDescriptor.of(PubsubMessage.class)).via(message -> {
  ...
})
```